src/chart-review/chart-review.css

```diff
@@ -55,3 +64,19 @@
 .topnav a {
   color: inherit;
 }
+
+.leftHeaderPagination {
+  padding: 0;
+  background-color: white;
+  cursor: pointer;
+  border: 0;
+  border-right: 1px solid grey !important;
+}
+
+.rightHeaderPagination {
```

Comment:
This is quite a repetition from `leftHeaderPagination`. Maybe make  rule `.leftHeaderPagination, .rightHeaderPagination { ... }` that contains the shared part?